tests/api/auth/test_auth_forbidden.py

```diff
@@ -0,0 +1,18 @@
+from . import UsspClient
+from . import is_client_response_forbidden
+from . import OirMocks
+
+
+class TestClassUssForbiddenAuthentication:
+    def __get_ussp_client(self) -> UsspClient:
+        return UsspClient(is_mocked=True)
+
+    def test_case_bearer_token_validation(self) -> None:
+        ussp_client = self.__get_ussp_client()
+
+        response = ussp_client.get_oir_by_id(OirMocks.OIR_ID)
+        if is_client_response_forbidden(response):
+            assert True
+            return
```

Comment:
This assertion could be made directly, eliminating the need for the `return` and the ending `assert False`
`assert is_client_reponse_forbidden(response) == True`

This applies for all the other files

Reply:
Done ✅